cypress/integration/pages/testsForAllCanonicalPages.js

```diff
@@ -1,7 +1,27 @@
 import envConfig from '../../support/config/envs';
+import config from '../../support/config/services';
+
+const serviceIsGNL = service => service === 'japanese';
 
 const testsForAllCanonicalPages = ({ service, pageType }) => {
   if (pageType !== 'errorPage404') {
+    describe('ATI', () => {
+      it('should have a noscript tag with an 1px image with the ati url', () => {
+        if (service === serviceIsGNL) {
```

Comment:
Same for the statement here - removing the nested ternary and having a single if/elseif/else statement